### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by frob**
Added optional alternate ports for nginx to aid in proxy pass deployments without needing to edit docker-compose file.

To deploy with alternate ports, add a `.env` file with the following contents:

```
OPENEDX_NGINX_PORT=8080
OPENEDX_NGINX_TLSPORT=8443
```

The .env file should not be added to the repository.

### CODE REVIEW

**Comment by frob** [comment]
This is to aid the issues brought up in #69

File: docker-compose.yml [hunk 1]
```diff
@@ -41,8 +41,8 @@ services:
     image: nginx:1.13
     restart: unless-stopped
     ports:
-      - "80:80"
-      - "443:443"
+      - "${OPENEDX_NGINX_PORT:-80}:80"
+      - "${OPENEDX_NGINX_TLSPORT:-443}443:443"
```
Comment (regisb) [inline docker-compose.yml]: Can I suggest naming the variables `NGINX_HTTP_PORT` and  `NGINX_HTTPS_PORT`? We don't need the `OPENEDX` prefix, as we are inside the openedx-docker repo, and HTTPS is more understandable (IMHO) than TLS.

Also, there is a typo in the 443 declaration. It should be:

    - "${OPENEDX_NGINX_TLSPORT:-443}:443"

**Comment by regisb** [comment]
This is a great idea, thanks @frob! We haven't used the docker-compose `.env` file for now, but it could be useful in many more ways.

File: docker-compose.yml (hunk 1 above)
Comment (frob) [reply docker-compose.yml]: Done

**Comment by frob** [comment]
> it could be useful in many more ways.

My thoughts exactly.

**Comment by regisb** [comment]
Merged in 2f9766a3a193707be846520a535a70b070531610
Thanks @frob for working on this!

**Pull request closed without merging**